src/cli/server/mod.rs

```diff
@@ -16,6 +17,15 @@ fn log_launch(sc: &ServerConfig) {
     );
 
     let url = sc.graphiql_url();
-    let url = format!("https://tailcall.run/playground/?u={}/graphql", url);
+    let utm_source = if VERSION.eq("0.1.0-dev") {
```

Comment:
Can you create a struct `Version` 

```rust
// version.rs

const DEFAULT_VERSION: &str = "0.1.0-dev";
pub const VERSION: Version = match option_env!("APP_VERSION") {
    Some(version) => Version { version },
    _ => Version{version: DEFAULT_VERSION},
};

pub struct Version {
  version: &'static str
}

impl Version {
  fn as_str() -> &'static str {
     self.version
  }
  fn is_dev() -> bool {
     self.version.eq(DEFAULT_VERSION)
  }
}
```

Something like above so that we don't keep checking for `0.1.0-dev` everywhere.